libraries/AP_Mount/AP_Mount_Backend.cpp

```diff
@@ -572,6 +581,41 @@ bool AP_Mount_Backend::get_angle_target_to_sysid(MountTarget& angle_rad) const
     return get_angle_target_to_location(_target_sysid_location, angle_rad);
 }
 
+// accessors for scripting backends
+bool AP_Mount_Backend::get_rate_target(float& roll_degs, float& pitch_degs, float& yaw_degs, bool& yaw_is_earth_frame)
```

Comment:
let's rename the last argument to "yaw_is_ef" to be consistent with other argument names.